<!--
The {#rmd-basics} text after the chapter declaration will allow us to link throughout the document back to the beginning of Chapter 1.  These labels will automatically be generated (if not specified) by changing the spaces to hyphens and capital letters to lowercase.  Look for the reference to this label at the beginning of Chapter 2.
-->
```{r include_packages_2, include = FALSE}
# This chunk ensures that the spartanodown package is
# installed and loaded. This spartanodown package includes
# the template files for the thesis and also two functions
# used for labeling and referencing
if(!require(devtools))
  install.packages("devtools", repos = "http://cran.rstudio.com")
if(!require(tidyverse))
    install.packages("tidyverse", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("ggplot2", repos = "http://cran.rstudio.com")
if(!require(bookdown))
    install.packages("bookdown", repos = "http://cran.rstudio.com")
if(!require(spartanodown)){library(devtools)
  devtools::install_github("ashley-williams/spartanodown")
  }
library(spartanodown)
library(tidyverse)
library(bookdown)
library(kableExtra)
library(knitr)
library(ggpubr)
library(ggplot2)
knitr::opts_chunk$set(echo=FALSE, warning = FALSE, message = FALSE, cache = FALSE)
```

# RESPONSE OF MERCURY METHYLATING BACTERIA TO THE COAL ASH SPILL IN THE DAN RIVER {#pcr}

## Introduction

The February 2014 coal ash spill mobilized coal ash into the Dan River. One constituent of particular concern during leaching and/or impoundment failure is mercury. Inorganic mercury often passes through an organism, but the bioavailable form of mercury, methylmercury (MeHg) is a known neurotoxin and potential endocrine disruptor and has a high affinity for sulfhydryl groups in proteins. This may destabilize proteins and lead to decreased enzymatic activity and reduced overall fitness of organisms[@driscollMercuryGlobalPollutant2013; @ehrlichGeomicrobiologyFifthEdition2008]. In submerged anoxic sediments under certain conditions, inorganic mercury ($\mathrm{Hg^{2+}}$) can be converted into MeHg through microbial metabolism [@dashBioremediationPotentialMercury2014; @schaeferActiveTransportSubstrate2011]. MeHg bioaccumulates and biomagnifies in the river food webs, posing a health risk to local residents who consume fish. [@dashBioremediationPotentialMercury2014; @otterTrophicStatusMetal2012; @roweBioaccumulationEffectsMetals2014]. The total available amount of MeHg within an ecosystem is controlled by multiple microbial and abiotic processes that reduce availability of $\mathrm{Hg^{2+}}$ or degradation of MeHg. $\mathrm{Hg^{2+}}$ can be volatilized as $\mathrm{Hg^{0}}$ through photoreduction or by bacteria with the *merA* gene [@boydMercuryResistanceOperon2012]. Additionally, MeHg can be demethylated into $\mathrm{Hg^{2+}}$ by sunlight [@tsuiPhotodegradationMethylmercuryStream2013] or microbes with the *merB* gene [@bizilyPhytoremediationMethylmercuryPollution1999]. 

Microorganisms have developed various mechanisms to mitigate effects of high concentrations of heavy metal toxins. These include reduction of the metal to a less toxic form, metal complexation, efflux pumps via an energy-dependent membrane transporter, and extracellular sequestration [@binkleyHeavyMetalsWastewater2003; @poulainCrackingMercuryMethylation2013]. MeHg is produced in anaerobic conditions predominately by sulfate reducing bacteria (SRB), iron reducing bacteria (FeRB), and methanogens [@liuAnalysisMicrobialCommunity2014].  Coal ash may provide the necessary substrates to stimulate the microbial methylation of Hg [@deonarineEnvironmentalImpactsTennessee2013]. 

Two genes are required for methylation of Hg, *hgcA* and *hgcB*.  As $\mathrm{Hg^{2+}}$ enters the cell, a methylated-HgcA protein transfers a -CH~3~ group to $\mathrm{Hg^{2+}}$ within the cytosol.  HgcB protein is then required to recycle the methylated-HgcA protein [@poulainCrackingMercuryMethylation2013]. The *hgcAB* sequence is conserved across multiple genera and therefore could be utilized as a molecular biomarker for suspected contaminated sites with real-time quantitative PCR [@christensenDevelopmentValidationBroadRange2016; @dashBioremediationPotentialMercury2014; @limadesilvaHeavyMetalTolerance2012]. Liu et al. (2014) found that the *hgcA* abundance and the concentration of MeHg in rice paddy soil in China near a mercury mining area is positively correlated [@liuAnalysisMicrobialCommunity2014]. This finding suggests that microbes may be contributing to the MeHg in the sampled soils. They also found high genetic diversity within the microbial community and that environmental factors such as total Hg, $\mathrm{SO_4}$, $\mathrm{NH_4}$, and organic matter influenced the community structure.  After phylogenetic analysis, the representative taxa in the community consisted of *Deltaproteobacteria*, *Firmicutes*, *Chloroflexi*, *Euryarchaeota*, and two novel taxa [@liuAnalysisMicrobialCommunity2014].

In 2008, a dike failure at the Tennessee Valley Authority Kingston Fossil Plant coal ash pond in Harriman, Tennessee, released an estimated 5.4 million cubic yards of ash into the surrounding community and rivers [@ruhlEnvironmentalImpactsCoal2010]. The release ruptured a natural gas line, disrupted power and transportation, destroyed three homes, and resulted in the evacuation of nearby neighborhoods. The impoundment pond has since been rebuilt and reinforced to resist natural disasters including earthquakes [@tvaFactSheetKingston2011]. In sediment samples collected downstream following the spill, total mercury concentrations were three to four times greater than sediments upstream of the spill. MeHg was also slightly higher than upstream [@deonarineEnvironmentalImpactsTennessee2013]. 

The coal ash spill into the Dan River similarly mobilized heavy metals into the environment. The extent of long-term effects of potential introduction methylated mercury into the food web of the river is unknown. Mercury, along with other coal ash constituents may stimulate mercury-methylating microorganisms in anaerobic sediments.  The goal of this study was to characterize the response of key microbial community constituents, specifically, *hgcA* abundance as a result of the Dan River coal ash spill. 

### Objective and hypothesis:
**Determine the spatial distribution of mercury-methylating taxa as a result of the coal ash spill using qPCR.** 
I hypothesize that there will be increased abundance of the in the SSU rDNA of mercury methylating taxonomic groups and the *hgcA* gene downstream of spill site due to stimulation by coal ash constituents present in the sediment. 

## Methods
### Study Sites and Sediment Collection
The Dan River is a 344 km river that rises in Patrick Co. Virginia and crosses into North Carolina in Stokes County. It flows across the border between NC and VA several times before flowing into the Kerr Reservoir on the Roanoke River which then flows to the Atlantic Ocean at the Albemarle Sound in North Carolina. This study encompasses sites up to 3.6 km upstream of the spill site in Eden, NC and 64.7 km downstream to Milton, NC, sampled in July 2015, about 17 months following the spill. 

To characterize the extent of the coal ash spill impact on the microbial community, samples were collected at three upstream reference sites, one site parallel to the ash ponds but upstream of the spill (leaching site), and five downstream sites including near two sites that were dredged for remediation, one at Town Creek, near the spill site and one near Abreu-Grogan Park, Danville, Va., and depositional sites that were not dredged near Danville (Figure \@ref(fig:map), Table \@ref(tab:sites)). 
\begin{center}
\end{center}
\begin{figure}[h]
\centering
    \includegraphics[width=400pt]{figure/map}
  \caption[Google Earth image of Dan River and sampling locations.]{Google Earth image of Dan River and sampling sites. Markers indicate sampling sites: Blue, upstream of spill site; Green, leaching site; Yellow, downstream dredged locations; Orange, downstream not dredged.}\label{fig:map}
\end{figure}


Each site was accessed by boat, where sediment from the riverbank and channel was collected. Riverbank sediment cores were collected in triplicate using a piston-style coring device. Channel samples were collected using a small dredge. Sediment cores were sectioned by depth and individual segments homogenized according to one of three sampling schemes to reduce the total number of samples to be assayed (Table \@ref(tab:scheme)). 0.25 $\mathrm{cm^3}$ samples were preserved in CTAB (cetyltrimethylammonium bromide) for DNA extraction. Channel sediment was homogenized then 0.25 $\mathrm{cm^3}$ was preserved in CTAB. 


```{r sites, echo=FALSE, message=FALSE, warning=FALSE}
read_csv("data/sites.csv") %>% 
  kable(booktabs=T, caption = "Sampling Locations.", align = "c", linesep="") %>% 
  column_spec(c(1,4:6), width = "1.5cm") %>%
  column_spec(2, width = "1.5cm") #%>%
  #landscape()
```

\begin{center}
\end{center}

```{r scheme, echo=FALSE, message=FALSE, warning=FALSE}
library(readxl)
library(tidyverse)
scheme <- read_excel("data/pcrdatasheets.xlsx", 
    sheet = "scheme")
scheme %>%  kable(booktabs=T, caption = "Sediment sampling schemes.", align = "c", linesep="") %>% 
  column_spec(2, width = "10cm")#%>% landscape()
```

### DNA extraction and qPCR

A CTAB extraction of DNA was performed using standard protocol for each sample [@stewartRapidCTABDNA1993]. The DNA extracted was quantified and subsequently diluted to a standard concentration of 5 ng/L in TE (Tris-EDTA) buffer pH 8.0 and stored at 4ºC. Extracted DNA quantity and purity were determined from 2 µL subsamples of each extraction using Thermo Scientific Nanodrop Spectrophotometer based on the 260/280 nm wavelength ratio.

An Applied Biosystems StepOne real-time PCR System was utilized to detect, amplify, and quantify target DNA gene proxies and representative taxa to meet the objective. Primers were chosen from literature targeting a general metabolic category. These include generic primers to the *hgcA* gene, a sulfate reducing gene, *dsr*, and the 16S rDNA of iron reducing bacteria. [@geetsDsrBGenebasedDGGE2006;@schaeferDetectionKeyHg2014;@wagnerPhylogenyDissimilatorySulfite1998;@dalyDevelopmentOligonucleotideProbes2000], Table \@ref(tab:primers). Each reaction contained the following: 10 µL of Power Sybr® Green PCR master mix, 1 µL of forward primer, 1 µL of reverse primer, 8 µL of sterile deionized water, and 1 µL of extracted DNA.  Three negative control reactions, samples repeated in triplicate, and positive standard controls serially diluted in triplicate, were ran in each 48 well plate. Extracted DNA from a pure culture of *Desulfovibrio africanus* (ATCC 19997), an isolate known to contain the *hgcA* gene was used as a standard for the *hgcA* and SRB targets. *Geobacter metallireducens* (ATCC 53774) served as the positive control and standard for FeRB primers. The real-time qPCR run method consisted of a holding stage for 5 minutes at 95ºC, followed by 40 cycles of 15 seconds at 95ºC, 1 minute at 57ºC, one minute at 72ºC, and a 15-second data collection at 80ºC, followed by a melt curve analysis beginning at 95ºC, dropping to 60ºC and increasing at 0.3ºC every second until it reaches 95ºC. 

The relative abundance of targets was computed by the StepOne software using the standard curve. The melt curve was examined to ensure amplicon specificity. 

The SSU rDNA abundance was normalized to the volume of extracted sediment as well as to organic matter. The amount of organic matter present in each sample was determined by ash free dry mass (AFDM). Each sample was dried at 60ºC for 48 hours, then combusted in a muffle furnace at 500ºC for 2 hours. Samples were weighed before and after combustion to calculate the AFDM. 

```{r primers, echo=FALSE, message=TRUE, warning=TRUE, results='asis'}
library(readxl)
library(kableExtra)
library(tidyverse)
df <- read_excel("data/pcrdatasheets.xlsx", sheet = "primers")
df[is.na(df)] <- " "
df %>% kable(booktabs=T, caption = "Primers used in this study.", align = "l",linesep = "" )
  

```

### Statistical Analysis
Statistical analyses included nonparametric tests for significance to determine the overall effect of sampling location on the abundance of the target SSU rDNA per sample. To determine whether abundance of the targets vary in the shore sediment based on the distance from the spill site, the data was first tested for normality with the Shapiro-Wilk test of normality. The data did not follow a Gaussian distribution, therefore, the data were transformed (LN(x +1)), but still failed the test for normality. Samples were binned according to location irrespective of depth due to limited replication of depths, and evaluated with nonparametric tests.  For each target, a Kruskall-Wallis test was performed to assess whether a difference in target SSU rDNA abundance exists between sites. Where a significant difference was observed, a Wilcoxon Rank Sum test was performed post hoc to determine differences between the location directly downstream of the spill site and all other locations independently. Target abundance of the leaching site was also compared to all other sites. These tests were then repeated with the data normalized to the mass of sediment and to the amount of organic matter present in each sample. 

Due to the lack of replication of river channel samples, the samples collected from the river channel bottom were binned into upstream and downstream groups. A Wilcoxon Rank Sum test was used to determine significance. All analyses were conducted using R [@rcoreteamLanguageEnvironmentStatistical2019]. 

## Results

### Shore Sediment Samples

The Kruskall-Wallis test revealed a significant difference between the abundance of amplified SSU rDNA and sampling locations for each of the three qPCR target assessed (Kruskall-Wallis p.value <0.05). Each of the targets amplified from the D-01 sampling site, the first sampling site downstream (0.3 km) from the spill site, was detected at a higher abundanc compared to at least one of the sites downstream and upstream (Figures \@ref(fig:hgcaomp) - \@ref(fig:srbomp), Table \@ref(tab:compare)). However, D-01 was not the location with the highest overall abundance of these targets. The sampling location U-03, the farthest upstream location, 3.6 km upstream of the spill, and 0.5 km downstream from the Smith River confluence, measured the hightest abundance of SSU rDNA for each of the targets tested (Figures \@ref(fig:hgcaomp) - \@ref(fig:srbomp), Table \@ref(tab:compareU)). 

Additionally, the leaching site, L-01 is not significantly different from the D-01 location. When comparing the leaching site to all others, targets abundance is slightly higher than most of the other sampling locations. This pattern is similar to the D-01 comparisons, but overall, less pairs are significantly different (Table \@ref(tab:compareL)). The same pattern was observed between the data normalized to the mass of sediment in each sample as well as the amount of organic matter present.    

```{r pcrdata, include=FALSE}

fontsize <- 8
ylab <- "SSU rDNA \U03BCg/g OM"
xlab <- "Sample Location"
sites <- read_csv("data/sites.csv")
ompcr <- read.csv("data/OMpcr.csv") %>% 
  gather(primer, concentration, HgcA:SRB, factor_key=TRUE) %>% 
  mutate(type = case_when(Scheme == "Dredge" ~ "Channel",
                          Scheme == "Ponar" ~ "Channel",
                          TRUE ~ "Shore"))
ompcr$Depth <- as.factor(as.double(ompcr$Depth))
ompcr$Sample <- factor(ompcr$Sample, levels = sites$`Site ID`)
hgcaom <- ompcr %>% filter(type=="Shore", primer == "HgcA")
hgcacompare <- compare_means(concentration ~ Sample, data = hgcaom, ref.group = "D-01") %>% mutate(target = "HgcA")
ferbom <- ompcr %>% filter(type=="Shore", primer == "FeRB")
ferbcompare <- compare_means(concentration ~ Sample, data = ferbom, ref.group = "D-01") %>% mutate(target = "FeRB") 
srbom <- ompcr %>% filter(type=="Shore", primer == "SRB")
srbcompare <- compare_means(concentration ~ Sample, data = srbom, ref.group = "D-01") %>% mutate(target = "SRB") 
```


\begin{center}
  \includegraphics[width=400pt]{figure/hgcaomp}
   \captionof{figure}[\textit{hgcA} abundance normalized to organic matter at each sampling location.]{\textit{hgcA} abundance normalized to organic matter at each sampling location. Asterisks signify difference between the reference, D-01 using the Wilcoxon Rank Sum test. * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001}\label{fig:hgcaomp}
\end{center}

\begin{center}
  \includegraphics[width=400pt]{figure/ferbomp}
   \captionof{figure}[FeRB abundance normalized to organic matter at each sampling location.]{FeRB abundance normalized to organic matter at each sampling location. Asterisks signify difference between the reference, D-01 using the Wilcoxon Rank Sum test. * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001}\label{fig:ferbomp}
\end{center}


\begin{center}
  \includegraphics[width=400pt]{figure/srbomp}
   \captionof{figure}[SRB abundance normalized to organic matter at each sampling location.]{SRB abundance normalized to organic matter at each sampling location. Asterisks signify difference between the reference, D-01 using the Wilcoxon Rank Sum test. * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001}\label{fig:srbomp}
\end{center}


```{r compare, echo=FALSE}
order <- c("U-03", "U-02", "U-01", "L-01", "D-01", "D-02", "D-03", "D-04", "D-05", "D-06") 

allcompare <- rbind(hgcacompare, ferbcompare, srbcompare) %>%  
  select(-c(1,2,4,5,6,8)) %>% 
  spread(key=target, value = p.signif) %>% 
  mutate(group2 =  factor(group2, levels = order)) %>%
  arrange(group2) %>% 
  rename(Location = group2)
allcompare <- allcompare[c(1,3,2,4)] 
allcompare %>% kable(booktabs=T, linesep = "",
                     caption = "Comparisons of each sampling location to the D-01 sampling location directly downstream of the spill. Wilcoxon Rank Sum Test, ns: not significant, * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001", 
                     caption.short = "Comparisons of each sampling location to the first downstream sampling location.",
                     align = "l" )
```


```{r compareL, echo=FALSE}
hgcacompareL <- compare_means(concentration ~ Sample, data = hgcaom, ref.group = "L-01") %>% mutate(target = "HgcA") 
ferbcompareL <- compare_means(concentration ~ Sample, data = ferbom, ref.group = "L-01") %>% mutate(target = "FeRB") 
srbcompareL <- compare_means(concentration ~ Sample, data = srbom, ref.group = "L-01") %>% mutate(target = "SRB") 
allcompareL <- rbind(hgcacompareL, ferbcompareL, srbcompareL) %>%  select(-c(1,2,4,5,6,8)) %>% 
  spread(key=target, value = p.signif) %>%
  mutate(group2 =  factor(group2, levels = order)) %>%
  arrange(group2) %>% 
  rename(Location = group2)
allcompareL <- allcompareL[c(1,3,2,4)]
allcompare %>% kable(booktabs=T, linesep = "",
                     caption = "Comparisons of each sampling location to the leaching site sampling location. Wilcoxon Rank Sum Test, ns: not significant, * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001",
                     caption.short = "Comparisons of each sampling location to the leaching site sampling location.",
                     align = "l" )
```

```{r compareU, echo=FALSE}
hgcacompareU <- compare_means(concentration ~ Sample, data = hgcaom, ref.group = "U-03") %>% mutate(target = "HgcA") 
ferbcompareU <- compare_means(concentration ~ Sample, data = ferbom, ref.group = "U-03") %>% mutate(target = "FeRB") 
srbcompareU <- compare_means(concentration ~ Sample, data = srbom, ref.group = "U-03") %>% mutate(target = "SRB") 
allcompareU <- rbind(hgcacompareU, ferbcompareU, srbcompareU) %>%  select(-c(1,2,4,5,6,8)) %>% 
  spread(key=target, value = p.signif) %>%
  mutate(group2 =  factor(group2, levels = order)) %>%
  arrange(group2) %>% 
  rename(Location = group2)
allcompareU <- allcompareU[c(1,3,2,4)]
allcompare %>% kable(booktabs=T, linesep = "", 
                     caption = "Comparisons of each sampling location to the furthest upstream sampling location. Wilcoxon Rank Sum Test, ns: not significant, * : p <= 0.05, ** : p <= 0.01, *** : p <= 0.001",
                     caption.short = "Comparisons of each sampling location to the furthest upstream sampling location.",
                     align = "l" )
```

### River Channel Samples

The samples collected in the river channel were limited by one sample at each site. To assess significance, samples were binned according to upstream and downstream. For each of the SSU rDNA targets tested, there is no significant difference between the groups (Figure \@ref(fig:channel)). 

\begin{center}
\end{center}
\begin{figure}

  \includegraphics[width=400pt]{figure/channel}
  \captionof{figure}[River channel abundance at sampling sites upstream and downstream.]{River channel abundance at sampling sites upstream and downstream normalized to organic matter.}\label{fig:channel}
\end{figure}

## Discussion
The SSU rDNA abundance detected at the sampling location 0.3 km downstream of the spill site, D-01, is higher than some locations upstream and downstream. However, the highest abundance of these three targets was found at the furthrest upstream site of all locations. Notably, the leaching site, L-01, and D-01 were not significantly different, and also exhibited higher target signal over some locations. This may signify that there is an impact from leaching of coal ash constituents. All primers tested exhibited a similar patterns of significance and abundance per location and may possibly be correlated due to an unknown confounding factor.

This field study was constrained by access with few boat ramps, and two dams. River gauge height also provided a logistical obstacle for repeated field sampling.  Additionally, the leaching site was accessed after permission from Duke Energy using their onsite boat ramp.  

Within this study, replication of core depth was limited, therefore, the interaction between the depth of samples was not evaluated. However, a scatterplot of the data did not show any clear patterns indicating a depth interaction, but the effect is inconclusive. We also experienced technical difficulties when assaying additional primer sets leading to a smaller suite of targets to determine overall microbial community response to the coal ash spill. 

The results from this microbial study are inconclusive. Our findings indicate that a significant difference exists between some of the sampling locations and the location directly downstream of the spill site, but the highest signal of SSU rDNA for each target was found at the furthest location upstream from the spill. Hg contamination within the sediment may still be possible, and our methods of sampling sediment may not have been robust to the variable and "hotspot" nature of the coal ash deposition and dispersal into the river sediments. It is plausible that the coal ash is buried in the sediment or has been transported further downstream than our furthest downstream location, 1.5 years after the spill event.    

