# CONCLUSIONS

The coal ash spill into the Dan River is the third worst spill in United States history. It released thousands of tons of ash and millions of gallons of untreated coal ash slurry into the Dan River. Approximately 1.5 years after the spill, the microbial response to the influx of coal ash into the sediment of the river was assessed using qPCR to probe samples targeting a mercury-methyating gene, and SSU rDNA from SRB and FeRB. Using these methods, relative to other locations, we found elevated abundance of  of the three targets assessed, adjacent to the ponds (leaching site) and the location directly downstream of the spill site. These results may indicate the taxa are responding to the coal ash, but we are unable to provide strong evidence using these methods. 

In this study we observed similar patterns of abundance of each target along the distance gradient. This indicates a similar response is detected between multiple taxonomic groups. Unexpectedly, the highest abundance among all targets was observed at a site that seems unlikely to have been contaminated with coal ash due to it's distance upstream from the spill site, where the coal ash was carried away downstream. We were unable to test if the target abundance is impacted by the depth of the sample due to inadequate replication, but we suspect any pattern within depth of sample may relate to the amount organic matter present in the sample, which could be highly variable.

Our sampling effort was constrained by multiple circumstances. The field sampling was limited by access to locations, weather, and river gauge height. We were also limited by the number of samples we were able to process. Additionally, the locations along the river available and selected for study may have been inadequate due to the unpredictability and variability of coal ash depositional sites, or "hotspots". These "hotspots" with a higher concentration of coal ash may stimulate more bacterial growth than other locations and thus provide a more definitive signal. Potentially, new technologies such as utilizing unmanned aerial vehicles to determine the size and scope of coal ash spills and other environmental pollution events, may aid in identifying hotspots and therefore, improve sampling strategies[@messingerUnmannedAerialVehicles2016]. 

Our results may also have been improved by assaying additional primers including primers designed to capture a broader suite of *hgcA* containing bacteria. Additionally, testing general bacterial primers to serve as a microbial abundance background to better characterize a possible correlation between the targets. 

We also characterized the metal tolerance of bacteria cultivated from coal ash. The observed tolerance among isolates is consistent with previous studies [@klubekCharacterizationMicrobialAbundance1992; @rajaCharacterizationBoronTolerant2012]. We only tested the growth of the isolates, and did not measure the ion concentrations directly to determine the if the metal was unbound and available for uptake. It is possible, the exposure level was lower than nominal concentrations [@hughesMetalSpeciationMicrobial1991; @ramamoorthyBindingMercuricOther1975]. 

The coal ash isolates and the soil isolates exhibited similar tolerances. This may indicate the bacteria isolated in the lab may be adapted to high levels of these metals where the soil isolates may be adapted to soil subjected to farming activities in the past. The isolates were identified by SSU rDNA sequencing and all were identified by genus, except one. The BLASTn search returned a sequence that has not been fully characterized for identification. Unsurprisingly, the population of culturable bacteria is represented by genera that are usually found in harsh environments.

The results of this study provide some insights to the response of bacteria to coal ash. If another natural disaster or pollution event were to occur, a similar study with improved sampling methods may help characterize the risk of mercury-methylating stimulation and mobilization into the food webs.      
