# CONCLUSIONS

The coal ash spill into the Dan River is the third worst spill in United States history. It released thousands of tons of ash and millions of gallons of untreated wastewater into the Dan River. Approximately 1.5 years after the spill, the microbial response to the influx of coal ash into the sediment of the river was assessed using qPCR to probe samples targeting a mercury-methyating gene and two taxonomic groups. Using these methods, relative to other locations, we found elevated abundance of SSU rDNA, of the three targets assessed, adjacent to the ponds (leaching site) and the location directly downstream of the spill site. These results may indicate a signal the taxa are responding to the coal ash, but we are unable to provide strong evidence using these methods. 

In this study we observed unexpected results. Similar patterns of SSU rDNA abundance of each target along the distance gradient may be the result of a confounding factor. The highest abundance of SSU rDNA in all qPCR reactions was found at a site that seems unlikely to have been contaminated with coal ash and may warrant further study. Also, we were unable to ascertain if the abundance of SSU rDNA is impacted by the depth of the sample, more statistical power could have been observed with more sampling at each depth.

The field sampling was limited by access to locations and the unpredictability and variability of coal ash depositional sites, or "hotspots". These "hotspots" with a higher concentration of coal ash may stimulate more bacterial growth than other locations and thus a more definitive signal. Potentially, new technologies such as utilizing unmanned aerial vehicles to determine the size and scope of coal ash spills and other environmental pollution events, may aid in identifying hotspots [@messingerUnmannedAerialVehicles2016]. 

Our results could have been bolstered by assaying additional primers including primers designed to capture a broader suite of *hgcA* containing bacteria. Additionally, testing general bacterial primers may elucidate any confounding factors by serving as a background control. 

We also characterized the metal tolerance of bacteria cultivated from coal ash.The observed tolerance among isolates is consistent with previous studies [@klubekCharacterizationMicrobialAbundance1992; @rajaCharacterizationBoronTolerant2012]. We only tested the growth growth of the isolates, and did not measure the ion concentrations directly to determine the if the metal was unbound and available for uptake. It is possible, the exposure level was lower than nominal concentrations. The coal ash isolates and the soil isolates exhibited similar tolerances, but this may indicate that the concentrations of some metals tested may not be high enough to elicit reduced growth. The isolates were identified by SSU rDNA sequencing and one previously unidentified bacteria was found. Unsurprisingly, the population of culturable bacteria is represented by genera that are usually found in harsh environments.

The methods employed in this study provided some insights to the response of bacteria to coal ash. If another natural disaster or pollution event were to occur, a similar study with the afore mentioned parameters may help characterize the risk of mercury-methylating stimulation and mobilization into the food webs.      
